elastic_fields/elastic_fields.go

```diff
@@ -0,0 +1,8 @@
+package elastic_fields
```

Comment:
```suggestion
package fields_name
```

and should be a pkg under the consts pkg

Reply:
I moved it under const pkg, but IMO we should leave the name to be "elastic_fields", because I have other pkg called "query_fields", and I don't want them to be confused. however I can merge them both into one pkg called fields_name if you think it's a better option.

Reply:
It is better IMO